### ISSUE TITLE
Failed: Install Docker Compose bash completion (Error 404: Not Found)

### ISSUE THREAD

**Issue by postfriday** [igor-nikiforov/ansible-role-docker#4]
@jidckii Hi!

There is an error while installing Docker Compose bash completion.

```
TASK [igor_nikiforov.docker : Install Docker Compose bash completion.] ***********************************************************************************************************
fatal: [alfa]: FAILED! => {"changed": false, "dest": "/etc/bash_completion.d/docker-compose", "elapsed": 0, "msg": "Request failed", "response": "HTTP Error 404: Not Found", "status_code": 404, "url": "https://raw.githubusercontent.com/docker/cli/master/contrib/completion/bash/docker-compose"}
```

My host:

```shell
root@vm1272769:~# lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.3 LTS
Release:	20.04
Codename:	focal
```

```shell
root@vm1272769:~# docker-compose version
docker-compose version 1.29.2, build unknown
docker-py version: 5.0.3
CPython version: 3.8.10
OpenSSL version: OpenSSL 1.1.1f  31 Mar 2020
```

**Comment by igor-nikiforov** [comment]
@postfriday thank you for report. I'll release a fix for this tomorrow. Until I fix it, please ignore this using `docker_compose_bash_completion_install: false`.

**Comment by igor-nikiforov** [comment]
@postfriday I released v1.3.0 with fix for this issue.
